common/client-libs/validator-client/src/nymd/traits/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod vesting_execute_client;
```

Comment:
Don't forget about the license notices, I think we have them everywhere in our main repo at this point